paranormal-pioneers/project/file_system/bin/ls.py

```diff
@@ -0,0 +1,94 @@
+from argparse import Namespace
+from datetime import datetime
+from typing import Iterator, List
+
+from project.core import command
+from project.core.parser import Parser
+from project.core.terminal import Terminal
+from project.core.utils import PathLike
+
+type_form = '<{}>'
+time_form = '({}; {})'
+size_form = '[{}]'
+name_form = '{}'
+
+
+class LS(command.Command):
+    """List all files in a given directory.
+    Example: ls -a -l ./bin
```

Comment:
I might be looking in the wrong place but where is the documentation for these options?